airflow/operators/branch_operator.py

```diff
@@ -0,0 +1,51 @@
+# -*- coding: utf-8 -*-
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+from airflow.models import BaseOperator, SkipMixin
+
+from typing import Union, Iterable, Dict
+
+
+class BranchOperator(BaseOperator, SkipMixin):
```

Comment:
```suggestion
class BaseBranchOperator(BaseOperator, SkipMixin):
```

Just to make it a tiny bit clearer that this is intended to be sub-classed (i.e. not used in a DAG, similar to BaseOperator)